tests/src/Search/Web/CacheWebTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace tests\eLife\Search\Web;
+
+class CacheWebTest extends ElasticTestCase
+{
+    public function testETag()
+    {
+        $this->addDocumentsToElasticSearch([
+            $this->getArticleFixture(0),
+            $this->getArticleFixture(1),
+        ]);
+        $this->newClient();
+        $this->jsonRequest('GET', '/search');
+        $response = $this->getResponse();
+        $this->assertEquals(200, $response->getStatusCode());
+        $eTag = $response->headers->get('ETag');
+        $this->assertEquals('max-age=300, public', $response->headers->get('Cache-Control'));
```

Comment:
Shouldn't this fail? `stale-while-revalidate` and `stale-if-error` should be there.